cmd/moby/trust_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
by "an array" I just mean you write the condition once and then pass it an array of eg
```
{
{true, "image", &TrustConfig{Image: []string{"image"}}
...
}
```
so to add more case you just add them to the array.